server/gfmd/user.c

```diff
@@ -1982,14 +1970,10 @@ gfm_server_user_info_get_by_auth_id(
 	/* XXX FIXME too long giant lock */
 	giant_lock();
 
-	e = gfarm_auth_user_id_type_from_name(auth_user_id_type_str,
```

Comment:
Please keep the type of first parameter of user_lookup_auth_id() as enum auth_user_id_type,
and recover the function call of  gfarm_auth_user_id_type_from_name().